geopandas/sindex.py

```diff
@@ -89,6 +74,28 @@ def __init__(self, geometry):
                 [None] * self.geometries.size, dtype=object
             )
 
+        @property
+        @classmethod
```

Comment:
Ughh this only works in >=3.9. I feel that it's important to make this a class property/method. Do we need it to be a property? Do we want to add our own decorator (we could build caching in as well, I think the whole thing is like ~30 LOC)? Open to ideas.

Reply:
Do not complicate things. This can be a simple property.

Reply:
Alrighty, I'll just make it a property then. I thought it'd be valuable to be able to retrieve this information without having to create a spatial index, but I admit it's not a big deal.

Reply:
Done